src/lib/dvf/token/getTokenInfo.js

```diff
@@ -1,5 +1,9 @@
 const bfxTodvf = require('dvf-utils').BfxToDvfToken
```

Comment:
Do we need bfxTodvf(token) ever?

I think in general we try to always use now: https://github.com/DeversiFi/dvf-client-js/blob/dev/src/lib/dvf/token/getTokenInfoOrThrow.js

And we want to deprecate getTokenInfo

Reply:
Modified parts of the code involved with this PR to use `getTokenInfoOrThrow.js` and added a mention on the `getTokenInfo`.
There are still plenty of files using it though, we might want to do a cleanup at some point.